packages/caravan-clients/src/client.ts

```diff
@@ -18,6 +18,7 @@ import {
   bitcoindWalletInfo,
 } from "./wallet";
 import BigNumber from "bignumber.js";
+import { Transaction } from "@caravan/health";
```

Comment:
I am moving the `Transaction` type to `@caravan-clients` also since UTXO type is coming from there itself. So Let's just have both of them at single source.